qa/replace-by-fee/README.md

```diff
@@ -0,0 +1,13 @@
+Replace-by-fee regression tests
+===============================
+
+First get version v0.5.0 of the python-bitcoinlib library. In this directory
+run:
+
+    git clone -n https://github.com/petertodd/python-bitcoinlib
```

Comment:
My thinking re: hard-coding is reproducability; if we're out of date, we should explicitly update it rather than implicitly.

Also, this set of tests will likely get removed fairly soon, and kept just for historical reference now that they've been mostly rewritten.